patched-vscode/extensions/sagemaker-idle-extension/README.md

```diff
@@ -0,0 +1,3 @@
+# Code Editor Idle Extension
```

Comment:
nit - updating naming here too, not blocking

Reply:
will update in follow up changes.